packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -61,25 +63,37 @@ export default class SelectNodeType extends PureComponent {
 
     componentDidMount() {
         this.setState({
+            filterSearchTerm: '',
             insertMode: calculateInitialMode(
                 this.props.allowedSiblingNodeTypes,
                 this.props.allowedChildNodeTypes
             )
         });
     }
 
+    updateInsertMode() {
```

Comment:
But wait, this method is only called from one place, why bother to create the abstraction?